src/library-authoring/create-library/data/api.ts

```diff
@@ -0,0 +1,28 @@
+import { camelCaseObject, getConfig } from '@edx/frontend-platform';
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+
+const getApiBaseUrl = () => getConfig().STUDIO_BASE_URL;
+
+/**
+ * Get the URL for creating a new library.
+ */
+export const getContentLibraryV2CreateApiUrl = () => `${getApiBaseUrl()}/api/libraries/v2/`;
+
+export interface CreateContentLibraryArgs {
+  title: string,
+  org: string,
+  slug: string,
+}
+
+/**
+ * Create a new library
+ */
+export async function createLibraryV2(data: CreateContentLibraryArgs) {
```

Comment:
Please define the return type for this function.

Reply:
Done: a7fe9997d255339cd72c1944071955250f78642d

Reply:
Thank you :)